data/json/items/gun/223.json

```diff
@@ -45,6 +45,29 @@
     "min_cycle_recoil": 1350,
     "magazines": [ [ "223", [ "stanag30", "stanag50", "survivor223mag" ] ] ]
   },
+  {
+    "id": "famas",
+    "copy-from": "rifle_auto",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "name": { "str": "FAMAS G2" },
+    "description": "An assault rifle that was until recently used by French armed forces.  Famous for its high rate of fire, it was never available in a civilian version.",
+    "weight": "3800 g",
+    "volume": "4674 ml",
```

Comment:
Two probs:
1) 
![image](https://user-images.githubusercontent.com/51099123/76359501-b9ad3c80-62f1-11ea-8b3f-3952005c328b.png)
44*93475.476/1000 to convert to ml= 4113.
2) It's supposed to be the displacement volume _without_ the magazine loaded, since that's defined by the mag volume(STANAG30rnd is 290ml per [Tonk math](https://github.com/CleverRaven/Cataclysm-DDA/pull/36838#discussion_r364904992)) minus magwell volume.

Reply:
1) Not sure why the calculation was off there, might have entered something wrong.
2) Right. It was pretty late when I did the FAMAS, so I forgot to take out the magazine.
3) Having handled one before, likewise having handled a KSG, and estimated how much thinner the FAMAS seemed. I could not find the width for any comparable rifle I could think of. If you can find a closer approximation I'll happily change it.